### DISCUSSION TITLE
Is there some dataset manipulation function?

### DISCUSSION THREAD

**Discussion by deniscsz** [pestphp/pest#469] [Q&A]
I have a dataset of postcode and it has been used in one test.

In other unit tests some function are tested passing a postcode as parameter. I would like to sample a single postcode from the previous dataset instead of use it hardcoded in my tests's codes.

Is there some dataset manipulation function to sample or get a specific item from the dataset?

Right now I'm doing it as below but since `Datasets` is marked as `internal`, it isn't "correct"
```
$dataset = \Pest\Datasets::get('postcode');
$postcode = $dataset[\array_rand($dataset, 1)];
```

**Comment by fabio-ivona** [comment]
@deniscsz we are discussing a possible solution in #470 :wink:
